TF_Application/Assets/TouchFree_Application/Scripts/TouchFreeMain.cs

```diff
@@ -48,5 +58,39 @@ public static void UpdateCursorWindowSize(float _cursorSize)
             CursorWindowSize = newWindowSize;
             CursorWindowMiddle = Vector2.one * (CursorWindowSize / 2);
         }
+
+        IEnumerator CheckForSettingsOpen()
+        {
+            WaitForSeconds wait = new WaitForSeconds(1);
+
+            yield return wait;
+
+            while (true)
+            {
+                IntPtr hwnd = FindWindow(null, "TouchFreeService");
```

Comment:
This name will need to change after TF-193

Reply:
I added a note to the miro board for this :)